numpyro/distributions/continuous.py

```diff
@@ -1033,6 +1033,33 @@ def icdf(self, q):
         return self.loc + self.scale * logit(q)
 
 
+class LogUniform(TransformedDistribution):
+    arg_constraints = {"low": constraints.positive, "high": constraints.positive}
+    support = constraints.positive
+    reparametrized_params = ["low", "high"]
+
+    def __init__(self, low=1., high=10., *, validate_args=None):
```

Comment:
I think 1. and 10. are a bit arbitrary. How about removing their default values?